mock/py/mockbuild/plugins/root_cache.py

```diff
@@ -127,6 +129,35 @@ def _unpack_root_cache(self):
         if self.rootCacheLock is None:
             self.rootCacheLock = open(os.path.join(self.rootSharedCachePath, "rootcache.lock"), "a+")
 
+        if self.buildroot.is_bootstrap and self.buildroot.use_bootstrap_image \
+                and not os.path.exists(self.rootCacheFile):
+            getLog().info("Using bootstrap image: %s" % self.buildroot.bootstrap_image)
```

Comment:
Use lazy evaluation. https://stackoverflow.com/questions/4148790/lazy-logger-message-string-evaluation

Reply:
Could you be more specific on how I should use the lazy evaluation?